.github/workflows/release.yaml

```
@@ -13,16 +13,15 @@ jobs:
       - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
         with:
           fetch-depth: 0
-          
-      - name: Set tag output
-        id: tag
-        run: echo "tag_name=${GITHUB_REF#refs/*/}" >> "$GITHUB_OUTPUT"
 
       - name: Install tejolote
         uses: kubernetes-sigs/release-actions/setup-tejolote@dd08496c83441d6477114cc0555b96d404dacff7 # v0.1.2
 
-      - run: |
-          tejolote attest --artifacts github://kubernetes-sigs/karpenter/${{ github.ref_name }} github://kubernetes-sigs/karpenter/"${{ github.run_id }}" --output karpenter.intoto.json --sign
+      - name: Run tejolote 
+        env:
+          TAG: ${{ github.ref_name }}
+        run: |
+          tejolote attest --artifacts github://kubernetes-sigs/karpenter/${TAG} github://kubernetes-sigs/karpenter/"${{ github.run_id }}" --output karpenter.intoto.json --sign
```

```suggestion
          tejolote attest --artifacts "github://kubernetes-sigs/karpenter/${TAG}" "github://kubernetes-sigs/karpenter/${{ github.run_id }}" --output karpenter.intoto.json --sign
```

Looks like you may still have globbing that can occur on this line due to not quoting the `TAG` environment variable